sway/tree/view.c

```diff
@@ -621,7 +637,16 @@ void view_unmap(struct sway_view *view) {
 
 	struct sway_seat *seat;
 	wl_list_for_each(seat, &input_manager->seats, link) {
-		cursor_send_pointer_motion(seat->cursor, 0, true);
+		if (config->mouse_warping == WARP_CONTAINER) {
+			struct sway_node *node = seat_get_focus(seat);
```

Comment:
It is possible for the seat to have no focus, or for focus to be a layer surface or xwayland unmanaged. In these cases `node` will be `NULL` which will produce a crash. You should be able to produce this by making a view unmap while the lockscreen is active.

Reply:
I'll add `node &&` to the conditional.